app/controllers/ticket_dispenser/tickets_controller.rb

```diff
@@ -3,7 +3,14 @@
 module TicketDispenser
   class TicketsController < ApplicationController
     def index
-      render json: Ticket.all.includes(:project, :owner, messages: :sender), status: :ok
+      query = Ticket.all.includes(:project, :owner, messages: :sender).order(:id)
+
+      created_after_date = ticket_params[:created_after] || 90.days.ago
```

Comment:
Instead of setting defaults, I think this would better if we only applied the extra `where` if the date scopes are provided.

The 90 day fallback is a potential gotcha later on, if someone assumes that index without any date params just returns all the data... which it will until you pass the 90 day mark.

Reply:
I'll open a ticket for this, it doesn't need to be done now.